### DISCUSSION TITLE
Do local version of files in Chirpy Starter overrides jekyll version of Chirpy?

### DISCUSSION THREAD

**Discussion by nyhryan** [cotes2020/jekyll-theme-chirpy#1609] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

For example, if I copy and paste `_sass/` directory from [jekyll-theme-chirpy](https://github.com/cotes2020/jekyll-theme-chirpy/tree/master/_sass) and put it locally inside my Chirpy Starter blog repository directory, are those files read first or override the version that is inside locally installed jekyll-theme-chirpy folder? (directory where you can find with command `bundle info --path jekyll-theme-chirpy`)

### Operations you have already tried

I have customized my blog's theme color by copying `_sass/` folder into my chirpy starter blog directory and editing `.scss` files there. I just want to clarify if my assumption is correct. (that local version of files that is also in jekyll-theme-chirpy directory, overrides same files in jekyll-theme-chirpy directory)

**Comment by kungfux** [comment, marked as answer]
Yes, in general, if you place a file with the same name and expected location that "hides" the original file from Chirpy, then it will be used to bundle your copy of the blog instead of the original.

I have not tried this specifically for files in the `_saas' folder, but what do you see when you run the blog locally? Does it work as you expect?

**Reply by nyhryan** [reply]
[My blog](https://nyhryan.github.io/portfolio/)
Yeah I customized some of scss files to change primary and some other colors into Catppuccin theme colors.

**Reply by kungfux** [reply]
I thought you did it some time back. It's hard to say for sure, maybe the colors were different. Looks great 👍

**Reply by nyhryan** [reply]
Oh I implemented this months ago haha, I am going to write about this in my jekyll blog tutorial post so I needed some confirmations on my theory.

**Reply by kungfux** [reply]
I see. This is a nice idea. Looking forward to reading it. Maybe I need to customize my blog colors too to make it more personalized 😀
